Source/Models/ChainpointResponse/ChainpointHashResponse.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+struct ChainpointHashResponse: Codable {
+    
+    struct Meta: Codable {
+        let submittedAt: String // Date
+        let processingHints: [String: String]
+        
+        enum CodingKeys: String, CodingKey {
+            case processingHints = "processing_hints"
```

Comment:
Should only conform to `Decodeable` and use `keycodingStrategy`
https://www.hackingwithswift.com/articles/52/swift-4-1-improves-codable-with-keydecodingstrategy